src/main/java/org/gradlex/javamodule/moduleinfo/ModuleInfo.java

```diff
@@ -112,6 +112,14 @@ public void ignoreServiceProvider(String ignoreServiceProvider) {
         addOrThrow(this.ignoreServiceProviders, ignoreServiceProvider);
     }
 
+    /**
+     * @param provider do not transfer service provider to the 'module-info.class'
+     * @param implementations the array of specific implementations to skip
+     */
+    public void ignoreServiceProvider(String provider, String... implementations) {
+        addOrThrow(this.ignoreServiceProviders, provider, implementations);
+    }
```

Comment:
Nitpick: as the second argument is `String...`, we could remove the `ignoreServiceProvider(String provider)` method above.

Reply:
Good point! Fixed. Just a habit to maintain binary compatibility but we don't need it here :-)